lib/template.html

```diff
@@ -38,10 +38,19 @@
             playerBrandingId: "pleaseFillTheRightPlayerId",
             skin: {
                 inline: {
-                    pauseScreen: {
-                        showDescription: false
-                    }
-                }
+					startScreen: {
```

Comment:
@DmitriiMaslov can we do away with these extra layers of indents here to keep the formatting clear?

Reply:
Formatting corrected